OpenOversight/app/main/model_view.py

```diff
@@ -117,20 +122,6 @@ def edit(self, obj_id, form=None):
 
         if not form:
             form = self.get_edit_form(obj)
-            # if the object doesn't have a creator id set it to current user
-            if (
-                hasattr(obj, "created_by")
-                and hasattr(form, "created_by")
-                and getattr(obj, "created_by")
-            ):
-                form.created_by.data = obj.created_by
-            elif hasattr(form, "created_by"):
-                form.created_by.data = current_user.get_id()
-
-            # if the object keeps track of who updated it last, set to current user
-            if hasattr(obj, "last_updated_by") and hasattr(form, "last_updated_by"):
-                form.last_updated_by.data = current_user.get_id()
-                form.last_updated_at.data = datetime.datetime.now()
```

Comment:
I think it would make sense to keep this here. That way we are updating the two columns for every edit.

```python
if hasattr(obj, "last_updated_by"):
    form.last_updated_by.data = current_user.get_id()
    form.last_updated_at.data = datetime.datetime.now()
```

Reply:
I've replaced this with the change in `populate_obj` which will update the `last_updated_by` and `last_updated_at` when editing objects